ampersand-state.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
Just enabling this along can allow JS runtimes to make some extra optimizations.